### PR TITLE
Add Gerrit Code Review news

### PR DESCRIPTION
**Pull request by lucamilanesio**

### CODE REVIEW

File: rev_news/drafts/edition-45.md [hunk 1]
```diff
@@ -124,8 +124,6 @@ This edition covers what happened during the month of October 2018.
 + Gerrit Code Review [2.16](https://www.gerritcodereview.com/2.16.html),
 [2.15.7](https://www.gerritcodereview.com/2.15.html#2157),
 [2.14.17](https://www.gerritcodereview.com/2.14.html#21417),
-[2.15.6](https://www.gerritcodereview.com/2.15.html#2156),
-[2.14.16](https://www.gerritcodereview.com/2.14.html#21416)
```
Comment (chriscool) [inline rev_news/drafts/edition-45.md]: I am ok with that, though we try to list all the recent releases even if they have been obsoleted. But I would suggest removing the comma at the end of the line related to 2.14.17, if you still want to remove the obsolete releases.

Comment (lucamilanesio) [reply rev_news/drafts/edition-45.md]: > I am ok with that, though we try to list all the recent releases even if they have been obsoleted. 

Sure, the problem is that 2.14.16 and 2.15.6 were completely broken (DB connection leak) and have been then dismissed. Better to remove any reference to them.

> But I would suggest removing the comma at the end of the line related to 2.14.17, if you still want to remove the obsolete releases.

Yes, let me do that now. Thanks.

**Comment by chriscool** [comment]
Thank you @lucamilanesio for the Gerrit User Summit 2018 report!

File: rev_news/drafts/edition-45.md (hunk 1 above)
Comment (lucamilanesio) [reply rev_news/drafts/edition-45.md]: Done.

Comment (chriscool) [reply rev_news/drafts/edition-45.md]: Great, thanks!

**Comment by chriscool** [comment]
I made a few improvements on top in 6b9c19f8cee90cc5785caf3d6adc1422b0930dd8.

**Comment by lucamilanesio** [comment]
Thanks a lot.

**Pull request merged**